virtualhost.sh

```diff
@@ -72,7 +72,7 @@ if [ "$action" == 'create' ]
 		<VirtualHost *:80>
 			ServerAdmin $email
 			ServerName $domain
-			ServerAlias $domain
+			ServerAlias www.$domain
```

Comment:
@lombax85 I think this should be optional, and in the case that user wants to include it, it should enables both alias (with/without www) and not only one.